less/notificationdrawer.less

```diff
@@ -23,19 +23,26 @@
     }
 
     .panel-collapse.in {
-      display: flex;
-      flex-direction: column;
-      flex: 1 1 auto;
       min-height: 0;
+      overflow-y: auto;
 
       .panel-body {
-        flex: 1 1 auto;
-        overflow-y: auto;
+        display: block;
+        .dropdown-menu-right {
+          position: absolute;
+        }
       }
     }
   }
 
   .drawer-pf-action {
-    flex: none;
+    position: sticky;
+    position: -webkit-sticky;
+    position: -moz-sticky;
+    position: -ms-sticky;
+    position: -o-sticky;
```

Comment:
Unfortunately, `position: sticky` is not supported in IE 11. 

@jeff-phillips-18 Do we need to include all of the vendor prefixes in this repo? Or do we use [autoprefixer](http://codeguide.patternfly.org/#css-prefixed-properties) as we do in patternfly core?

Reply:
I don't believe we currently use autoprefixer.